<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManagerClass</class>
 <widget class="QMainWindow" name="DeviceManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>771</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>DeviceManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="vl_UI">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="top_wgt" native="true">
      <property name="minimumSize">
       <size>
        <width>1014</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(&quot;:/DeviceManager/topbar&quot;)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:&quot;transpant&quot;</string>
         </property>
         <widget class="QLabel" name="title">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>40</y>
            <width>291</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:&quot;transpant&quot;;font-size:36px;color:white</string>
          </property>
          <property name="text">
           <string>教学管理软件</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="right_wgt" native="true">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image:&quot;transpant&quot;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="4">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="0">
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>100</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="7">
                 <widget class="QPushButton" name="pb_close">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="toolTip">
                   <string>关闭</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image: url(:/DeviceManager/images/关闭-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
	background-image: url(:/DeviceManager/images/关闭-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/DeviceManager/images/关闭-默认.png);
    border:0px;
}

</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="rtlabel">
                  <property name="minimumSize">
                   <size>
                    <width>69</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Algerian</family>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:#98b8f9</string>
                  </property>
                  <property name="text">
                   <string>当前型号</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="6">
                 <widget class="QPushButton" name="pb_min">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="toolTip">
                   <string>最小化</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton:hover{
	background-color:transparent;
	background-image:url(:/DeviceManager/images/最小化-悬浮.png);
    border:0px;}
QPushButton:pressed{
	background-color:transparent; 
    background-image: url(:/DeviceManager/images/最小化-点击.png);
    border:0px;}
QPushButton {
	background-color:transparent;
	background-image: url(:/DeviceManager/images/最小化-默认.png);
    border:0px;
}

</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="ctlabel">
                  <property name="font">
                   <font>
                    <family>Algerian</family>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:#98b8f9</string>
                  </property>
                  <property name="text">
                   <string>北京时间</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="5">
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>30</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>100</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>36</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="curtime">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>36</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>36</height>
                   </size>
                  </property>
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>255</red>
                        <green>255</green>
                        <blue>255</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="WindowText">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>120</red>
                        <green>120</green>
                        <blue>120</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Button">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>120</red>
                        <green>120</green>
                        <blue>120</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>56</red>
                        <green>118</green>
                        <blue>234</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="font">
                   <font>
                    <family>Algerian</family>
                    <pointsize>18</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:rgba(56,118,234,1)</string>
                  </property>
                  <property name="text">
                   <string>2011-12-23 12:12:12</string>
                  </property>
                  <property name="textFormat">
                   <enum>Qt::PlainText</enum>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_6" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>10</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>10</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_min_1">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="toolTip">
                   <string>当前用户</string>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	background-color:transparent;
	background-image: url(:/DeviceManager/images/用户1.png);
    border:0px;
}

</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lb_user">
                  <property name="styleSheet">
                   <string notr="true">color:white;font-size:14px;font-family:&quot;Microsoft Yahei&quot;
</string>
                  </property>
                  <property name="text">
                   <string>admin</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap resource="DeviceManager.qrc">:/DeviceManager/xiala</pixmap>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="center_wgt" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <widget class="QWidget" name="tab_DeviceManagement">
          <attribute name="title">
           <string>任务模型配置</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_faultInjection">
          <attribute name="title">
           <string>故障模型配置</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_Display">
          <attribute name="title">
           <string>知识库配置</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_taskManege">
          <attribute name="title">
           <string>任务岗位配置</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="top_wgt_2" native="true">
      <property name="minimumSize">
       <size>
        <width>1014</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> 
background-color: rgb(85, 170, 255);</string>
      </property>
      <widget class="QWidget" name="wgt_status_left" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>790</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>790</width>
         <height>24</height>
        </size>
       </property>
       <widget class="QLabel" name="lb_flag_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>4</y>
          <width>24</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="DeviceManager.qrc">:/DeviceManager/images/icon_info.png</pixmap>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DeviceManager.qrc"/>
  <include location="FaultInjection.qrc"/>
 </resources>
 <connections/>
</ui>
